### PR TITLE
Added Laravel 6

### PR DESCRIPTION
**Pull request by pablo-bossi**
Hi, I'm using some of your libraries, and now I am on the process to upgrade to Laravel 6 (Eventually to Laravel 7), so found out there isn't yet support for laravel 6 so I'm building it myself.
So will be sending a few PR's for some of your libraries if that's ok.

Unit Tests ran successfully
Also tested this with a site that uses Bruno and other packages depending on architect and everything runs smoothly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,24 +22,24 @@ Book 1-----n Author
 This is how related resources are loaded by default using embedded mode.
 
 ```json
-{
```
Comment (pablo-bossi) [inline README.md:25]: Just removal of trailing spaces, sorry, only real change here is that for Laravel support a new version would be needed as some of the dependencies updated are not compatible with both Laravel 5 and 6

File: README.md [hunk 2]
```diff
@@ -133,9 +133,8 @@ $parsed = $architect->parseData($books, [
 nice convenience methods to define the Architect relationships in query parameters.
 
 ## Installation
-
 ```bash
-composer require optimus/architect ~1.0
+composer require optimus/architect dev-laravel-6
```
Comment (pablo-bossi) [inline README.md:137]: This is the only real change, I guess if you accept this PR, you'll name a new version didn't dare to decide that is V2.0 or so

File: composer.json [hunk 3]
```diff
@@ -6,13 +6,13 @@
         }
     },
     "require": {
-        "laravel/framework": "~5.1"
+        "laravel/framework": "~6.0"
```
Comment (pablo-bossi) [inline composer.json:9]: Use laravel 6 framework, removing 5.1 as some of the other required libraries are compatible either for laravel 5 or for laravel 6 but not both

File: composer.json [hunk 4]
```diff
@@ -6,13 +6,13 @@
...
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*",
-        "orchestra/testbench": "~3.1",
-        "phpunit/phpunit": "~4.7",
-        "satooshi/php-coveralls": "dev-master@dev"
+        "mockery/mockery": "1.3.*",
+        "orchestra/testbench": "4.*",
+        "phpunit/phpunit": "~8.0",
+        "php-coveralls/php-coveralls": "2.2.*"
```
Comment (pablo-bossi) [inline composer.json:15]: satooshi/php-coveralls got deprecated so updated to the suggested replacement.

File: composer.json [hunk 5]
```diff
@@ -6,13 +6,13 @@
...
+        "laravel/framework": "~6.0"
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*",
-        "orchestra/testbench": "~3.1",
-        "phpunit/phpunit": "~4.7",
-        "satooshi/php-coveralls": "dev-master@dev"
+        "mockery/mockery": "1.3.*",
+        "orchestra/testbench": "4.*",
+        "phpunit/phpunit": "~8.0",
```
Comment (pablo-bossi) [inline composer.json:14]: Updated dependencies supported for laravel 6.

File: phpunit.xml [hunk 6]
```diff
@@ -7,8 +7,7 @@
      convertNoticesToExceptions="true"
      convertWarningsToExceptions="true"
      processIsolation="false"
-     stopOnFailure="false"
-     syntaxCheck="false">
```
Comment (pablo-bossi) [inline phpunit.xml:11]: Remove syntaxCheck as it was throwing warnings with the new phpUnit version

**Comment by eenchev** [comment]
@esbenp could you please review that and cut new release?

**Comment by esbenp** [comment]
Thank you for this @pablo-bossi ! :-)

**Pull request merged**